front/app/modules/commercial/custom_maps/utils/map.ts

```diff
@@ -68,17 +72,11 @@ export const getTileProvider = (
 
 export const getTileOptions = (tileProvider?: string | null) => {
   if (tileProvider?.includes('maptiler')) {
-    return {
-      tileSize: 512,
-      zoomOffset: -1,
-      detectRetina: false,
-      minZoom: 1,
-      maxZoom: 19,
-      crossOrigin: true,
-      subdomains: ['a', 'b', 'c'],
-      attribution:
-        '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
-    };
+    return MAPTILER_DEFAULT_OPTIONS;
+  }
+
+  if (tileProvider?.includes('wien.gv.at/basemap')) {
```

Comment:
I find it risky to use `.includes?` to check if a string is equal to a string. If we have a tileProvider 'maptiler-2' or 'maptiler-xl' or ..., it'll also return true. How about we compare the string strictly? `tileProvider === 'wien.gv.at/basemap'` etc.

And while we're at it, we can type the tile providers. `type TTileProvider = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png' | ...`